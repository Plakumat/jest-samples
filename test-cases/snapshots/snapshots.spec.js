import user from "./snapshots"

test('without snapshots', () => {
    // Let's add the string version of the user Object
    const userString = "{\"name\":\"Tony Tinkerton\",\"age\":42,\"job\":\"inventor\"}"

    // and compare it accordingly
    expect(JSON.stringify(user)).toBe(userString)
})

test('user matches with snapshot', () => {
    expect(user).toMatchSnapshot()
})

//for update snapshot --> jest --updateSnapshot