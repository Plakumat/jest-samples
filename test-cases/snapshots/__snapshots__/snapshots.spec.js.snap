// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches with snapshot 1`] = `
Object {
  "age": 42,
  "job": "inventor",
  "name": "Tony Tinkerton",
}
`;
